docs/context/ref/ChatInitSettings.md

```diff
@@ -62,7 +62,21 @@ const initSettings = {
         allowHistoryBrowsing: true,
         allowMessageCopy: true
     }
-    initMessage: 'Hello both!'
+    message: {
+      type: 'fdc3.message',
+      text: {
+        'text/plain': 'Hey all, can we discuss the issue together? I attached a screenshot'
+      },
+      entities: {
+         '0': {
```

Comment:
update example to show use of string IDs? How will the string IDs be used/quoted in messages>

Reply:
I put a more detailed example in Message.md , using string IDs. For the reference of an ID from within a message, we're working on some suggestions to do this in a "markdown" way - will suggest later